skabase/SKALogger/setup.py

```diff
@@ -33,7 +35,7 @@
       entry_points={'console_scripts':['SKALogger = SKALogger:main']},
       author='cam',
       author_email='cam at ska.ac.za',
-      license='BSD-3-Clause',
```

Comment:
The license value is BSD-3-Clause.

Reply:
This is an outdated change. It's correct in current code.